Scripts/script-FetchFromInstance.yml

```diff
@@ -17,6 +19,8 @@ script: |
           contents = inc.get('Contents', '')
           if 'Error' in contents:
               raise Exception("Error occurred while fetching incidents from {0}: {1}".format(instance_name, contents))
+          if re.match("invalid character \'[a-zA-Z]\' looking for beginning of value", contents, re.IGNORECASE):
```

Comment:
Do we get this error only with letter characters ?

Reply:
As far as I can recall, we can always add a fix for that

Reply:
As far as I can recall, we can always add a fix for that